pages/[id]/bridge.tsx

```diff
@@ -68,6 +68,7 @@ const Bridge = () => {
   });
 
   const needsAllowanceL1 = (allowanceL1 || 0) < rawAmount;
+  const isSufficientBalance = (source.token?.value || 0) >= rawAmount;
```

Comment:
[arbitrary place to leave comment]

functionality currently not working when i have 0 balance. Asks me to set approval, no error displayed (although, red error icon displays in input box)

Reply:
https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/129/commits/1f438166e417b47324f54d81c9704c19638407b4